packages/components/src/InputField/InputField.purs

```diff
@@ -101,7 +102,7 @@ render self@{ props, state } =
             , className:
                 if isJust props.validationError
                 then "input-field--invalid-field"
```

Comment:
Doing it this way will remove all element specific classes in case there's a validation error.

Reply:
I fixed it per your suggestion :)